doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -257,20 +264,21 @@ Some operations imply that only a partial representation of an entity will be pr
 
 * `id` and `type` are not allowed in update operations, as they are inmutable properties.
 
-* In the requests in which entity `type` is allowed, it may be omitted. When omitted, the default string value `none`
-  is used for the type.
```

Comment:
This is something worth capturing on the release notes as under discussion and a potential hole for JSON-LD compatibility. To my mind an attribute, if defined, always has a type. If no explicit type is given, then its type will be the JSON data type, so probably the 'none' artefact is a good candidate for revisiting and removal